guides/installation/overview.md

```diff
@@ -54,7 +54,7 @@ Although Shopware 6 supports most UNIX like environments, we recommend using **U
 
 PHP
 
-* 7.4.3 or higher
+* tested on 7.4.3, 8.0 and 8.1
```

Comment:
Any other line (that starts with non-technical text) starts with an upper-case character. Should it be normalized?

Reply:
I agree it should be uniform